### DISCUSSION TITLE
yalt source code location

### DISCUSSION THREAD

**Discussion by galer7** [triggerdotdev/trigger.dev#1287] [Q&A]
Hello, I was reading this great [article](https://trigger.dev/changelog/yet-another-local-tunnel) you guys posted about your in-house tunneling solution, and I was wondering if it's still open source. The article points to [here](https://github.com/triggerdotdev/trigger.dev/tree/main/apps/yalt) but I get a 404 on the `main` branch.

**Comment by casyalex** [comment]
https://github.com/triggerdotdev/trigger.dev/pull/1265/commits/8b9caeea7ca5d55e2e292b5ba5f32f2885c1e277

They move it away from the main repo.
